### DISCUSSION TITLE
Realtime error:  net::ERR_HTTP2_PROTOCOL_ERROR 200 (OK)

### DISCUSSION THREAD

**Discussion by adamshand** [pocketbase/pocketbase#5252] [Q&A] [answered]
I'm getting the below error in the browser on a SvelteKit page which waits for the user to be verified before allowing them to progress to the app.  

Everything seems to be working as expected (before and after the error), but the error shows up after a few minutes and repeats periodically. I'm confused by the error and the `(OK)`.

There's no proxy in front of PB and I'm not seeing any other sporadic connection issues to PB.

Any suggestions or explanations would be appreciated?

I'm using the latest PB 0.22.17 and just upgraded to lastest js-sdk and svelte 5.

```
GET https://pb.xxx.nz/api/realtime net::ERR_HTTP2_PROTOCOL_ERROR 200 (OK)                               realtime:1 
GET https://pb.xxx.nz/api/realtime net::ERR_HTTP2_PROTOCOL_ERROR 200 (OK)                               realtime:1 
...
```

The relevant code is:

```
// src/routes/(auth)/verify/+page.svelte
<script>
	import { pb } from '$lib/pocketbase.svelte';
	import { page } from '$app/stores';
...
	$effect(() => {
		$page.data.user || goto('/');
		$page.data.user?.verified && goto('/app');

		pb.collection('users').subscribe($page.data.user?.id, (e) => {
			dev && console.log(`verify: action: ${e.action} verified=${e.record.verified}`);
			if (e.record.verified) {
				goto('/app');
			}
		});

		return () => {
			dev && console.log('verify: unsubscribe', $page.data.user?.id);
			pb.collection('users').unsubscribe($page.data.user?.id);
		};
	});
</script>
```

**Comment by ganigeorgiev** [comment, marked as answer]
The above just indicates that the connection was interrupted/dropped.
It is most likely caused by the default PocketBase 5mins idle timeout (_aka. no realtime messages were sent in that period_) to prevent forgotten/"zombie" clients (_depending where it is deployed some hosting services like fly.io may apply shorter connections limit_).
In any case usually such errors can be ignored as the SDK will autoreconnect behind the scenes (if the client is still alive).

**Reply by adamshand** [reply]
Great, thanks!
